README.md

```diff
@@ -350,8 +348,13 @@ Then targeting a Ninja make file:
 cmake -GNinja -DJSCORE_LIBRARY=/usr/lib/x86_64-linux-gnu/libjavascriptcoregtk-4.0.so ..
 ```
 
-Ninja is not mandatory and make can be used instead.
-And finaly, run a build:
+Ninja is not mandatory and make can be used instead:
```

Comment:
If `ninja-build` is a dependency, we should just tell people to use ninja to build. Having a fork at each step seems confusing to me.

Reply:
makes sense, I'll just remove the additional instruction.